pwndbg/commands/__init__.py

```diff
@@ -23,10 +23,26 @@
 
 commands = []
 
+def list_current_commands():
+    gdb.execute('set pagination off')
+    command_list = gdb.execute('help all', False, True).strip().split('\n')
+    existing_commands = set()
+    for line in command_list:
+        line = line.strip()
+        # Skip non-command entries
+        if len(line) == 0 or line.startswith('Command class:') or line.startswith('Unclassified commands'):
+            continue
+        command = line.split(' ')[0]
```

Comment:
Why not just `list.split()[0]` so we will split on *any whitespaces of any length*?

Reply:
Agree, will change